pkg/sessionctx/variable/sysvar.go

```diff
@@ -2615,6 +2615,11 @@ var defaultSysVars = []*SysVar{
 			s.OptOrderingIdxSelThresh = tidbOptFloat64(val, DefTiDBOptOrderingIdxSelThresh)
 			return nil
 		}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBOptOrderingIdxSelRatio, Value: strconv.FormatFloat(DefTiDBOptOrderingIdxSelRatio, 'f', -1, 64), Type: TypeFloat, MinValue: -1, MaxValue: 1,
```

Comment:
Could you explain what is `-1` and `1` means here?

Reply:
How about `-1 < Value < 0`?

Reply:
> How about `-1 < Value < 0`?

I have update the release notes to include < 0.
The default is that this enhancement is disabled, but users can control the optimism/pessimism associated with this estimation with variable tidb_opt_ordering_index_selectivity_ratio.
1. Value <  0: new estimation is disabled. Default = -1.
2. Value 0 : rows are estimated to be found immediately within the scan range.
3. Value >=0 and <= 1 : a percentage of the scan range when a row will be found. 0.1 means 10% of the scan range. 0.5 means 50%. 1 means 100%.

I have also updated the code (cross_estimation.go) and comments (in session.go) to account for this.